### DISCUSSION TITLE
Starter versus multiple module model

### DISCUSSION THREAD

**Discussion by yoshitamagoshi** [fivethree-team/nestjs-prisma-client-starter#456] [Q&A] [answered]
Hi folks, thanks very much for this excellent starter!

I was wondering if you could shed some light on the starters folder structure as apposed to the NestJS recommended module-first structure. I like the simplicity of the starters structure but hesitate to deviate from NestJS' recommendation (as well as what's generated by `nest g resource`):

> "While very small applications may theoretically have just the root module, this is not the typical case. We want to emphasize that modules are strongly recommended as an effective way to organize your components." ([Modules doc page](https://docs.nestjs.com/modules))

Thanks again!

**Comment by marcjulian** [comment]
Hi @yoshitamagoshi, thanks so much. Great question. We started implementing this starter before `nest g resource` came out, thus it has a different folder structure. I also prefer the structure produced by `nest g resource`, but haven't found the time to refactor the starter. Do you like to add a PR which uses the Nest recommendation?

**Comment by hymair** [comment, marked as answer]
Created a PR: https://github.com/notiz-dev/nestjs-prisma-starter/pull/460

**Reply by marcjulian** [reply]
PR is reviewed and merged! The starter now follows the recommended approach from NestJS.

Appreciate your contribution @hymair!
